uint8 id
uint32 system_time
uint8 pixel_count
TofsenseMFrame0Pixel[] pixels
